### ISSUE TITLE
How to choose right num classes for conditional batch normalization

### ISSUE THREAD

**Issue by odgiv** [proceduralia/pytorch-neural-enhance#1]
Hello, 
How did you choose values of nums_classes=(6,3,3,4) for conditional batchnorms?
Thanks.

**Comment by proceduralia** [comment]
Ciao odgiv,
the number of classes in multiple conditional batch normalization is determined by the categorical attributes of the MIT-Adobe FiveK dataset.
(6,4,3,3,4) corresponds to Subject, Light, Location and Time.

**Comment by proceduralia** [comment]
Ciao odgiv,
the number of classes in multiple conditional batch normalization is determined by the number of classes for each categorical attribute of the MIT-Adobe FiveK dataset.
They are the attributes representing the semantic information, namely: 
> indoor vs. outdoor, time of day, type of light, and main subject (people, nature, man-made objects, etc).

**Comment by odgiv** [comment]
Hello proceduralia,
Thank you for you response.
